src/udiskslinuxdrive.c

```diff
@@ -40,6 +40,7 @@
 #include <mntent.h>
 
 #include <glib/gstdio.h>
+#include <libstoragemgmt/libstoragemgmt.h>
```

Comment:
The function in this PR does not commonly used on laptop or workstation. We may add support of laptop/workstation disk LEDs using intel ledmon way, but no document found hence no plan yet.